### DISCUSSION TITLE
刚接触EMQX想了解一下gitbub源码包的不同？

### DISCUSSION THREAD

**Discussion by onejimmyboy** [emqx/emqx-operator#974] [General]
看了一下gitbub存在两种类型的安装包，想知道这两个包的却别，看了一下官方的介绍，没有找到具体的区别，
![image](https://github.com/emqx/emqx-operator/assets/57432151/2a19bf4c-cf24-48a4-946f-708f2a166f57)

**Comment by Rory-Z** [comment]
You can try 2.2.4, because it is `latest` release. EMQX operator follows the [semantic versioning specification](https://semver.org/), so you should try versions in the format of x.y.z.

**Reply by onejimmyboy** [reply]
上面的1.2.7-ecp的包代表啥意思呐

**Reply by Rory-Z** [reply]
Just patch for [EMQX ECP](https://www.emqx.com/en/products/emqx-ecp)
